src/views/InvolvementsAll/components/Requests/components/RequestSent/index.jsx

```diff
@@ -2,24 +2,24 @@ import { Button, Divider, Grid, Typography } from '@mui/material';
 import ClearIcon from '@mui/icons-material/Clear';
 import { formatDistanceToNow } from 'date-fns';
 import requestService from 'services/request';
-import { gordonColors } from 'theme';
 // @TODO CSSMODULES - outside directory
-import styles from '../../Requests.module.css';
+import styles from './RequestSent.module.css';
 
 const RequestSent = ({ member, onCancel }) => {
   const handleCancel = () => {
     requestService.cancelRequest(member.RequestID);
     onCancel(member); // Updates state of parent component to cause rerender
   };
 
-  const button = {
-    color: gordonColors.secondary.red,
-  };
-
   let cancel;
   if (member.Status === 'Pending') {
     cancel = (
-      <Button variant="outlined" size="small" style={button} onClick={handleCancel}>
+      <Button
+        variant="outlined"
+        size="small"
+        className={styles._cancel_button}
```

Comment:
You can probably use the color='error' here, and get rid of the CSS.  Just check what that looks like.